tests_e2e/pipeline/pipeline-cleanup.yml

```diff
@@ -4,7 +4,7 @@
 # Runs every 3 hours and deletes any resource groups that are more than a day old and contain string "lisa-WALinuxAgent-"
 #
 schedules:
-  - cron: "0 */3 * * *" # Run every 3 hours
+  - cron: "0 */12 * * *" # Run twice a day (every 12 hours)
```

Comment:
I decreased the frequency of cleanup. Twice a day should be enough, we can adjust later if needed.